packages/by-name/kata/contrast-node-installer-image/package.nix

```diff
@@ -91,6 +88,20 @@ let
               url = "file:///opt/edgeless/snp/share/qemu/efi-virtio.rom";
               path = "/opt/edgeless/@@runtimeName@@/snp/share/qemu/efi-virtio.rom";
             }
+            {
+              url = "file:///bin/nydus-overlayfs";
+              path = "/opt/edgeless/@@runtimeName@@/bin/nydus-overlayfs";
+              executable = true;
+            }
+            {
+              url = "file:///opt/edgeless/tdx/share/OVMF.fd";
+              path = "/opt/edgeless/@@runtimeName@@/tdx/share/OVMF.fd";
+            }
+            {
+              url = "file:///opt/edgeless/tdx/bin/qemu-system-x86_64";
+              path = "/opt/edgeless/@@runtimeName@@/tdx/bin/qemu-system-x86_64";
+              executable = true;
+            }
```

Comment:
It looks like this is just reordering items - is that intentional?

Reply:
No, this is a byproduct of splitting out the TDX-specific stuff and merging it again. Will adjust